orchestrators/ProcurementOperations/tests/Unit/Services/EarlyPaymentDiscountServiceTest.php

```
@@ -33,315 +30,148 @@ protected function setUp(): void
-        );
-
-        $this->assertInstanceOf(EarlyPaymentDiscountData::class, $discount);
-        $this->assertEquals('INV-001', $discount->invoiceId);
-        $this->assertEquals('VENDOR-001', $discount->vendorId);
-        $this->assertEquals(10000.00, $discount->invoiceAmount->getAmount());
-        $this->assertEquals(2.0, $discount->discountPercentage);
-        $this->assertEquals(10, $discount->discountDays);
-        $this->assertTrue($discount->isAvailable);
+        $discount = $this->service->getEarlyPaymentDiscount('tenant-1', 'vendor-1', 'inv-1');
+        $this->assertNull($discount);
     }
```

PHPMD (2.15.0)

[error] The method it_returns_null_for_get_early_payment_discount is not named in camelCase. (undefined)

(CamelCaseMethodName)